src/main/java/tempest/ui/ViewManager.java

```diff
@@ -46,8 +49,13 @@ public void changeView(String name) {
   public void changeToPrevious() {
```

Comment:
Should this not be in `changeView()`? Because, for example, if there are no modules when the program is started update() won't be run.